deployments/fluxcd/values.yaml

```diff
@@ -8,4 +8,19 @@ git:
 additionalArgs:
   - --manifest-generation=true
 syncGarbageCollection:
-  enabled: true
\ No newline at end of file
+  enabled: true
+resources:
```

Comment:
do we need this here? i thought we just did the minimum and then let flux config take over?

Reply:
sure